### PR TITLE
fix: add shivam-jha2712

### LINKED ISSUE #1412: Shivam Jha
## Profile Screenshot

Screenshot (1298)

### PR DESCRIPTION
**Pull request by shivam-jha2712**
resolve #1412

resolves #1412 

## Checklist for profile addition

- [x] Name the PR `fix: add shivam-jha2712`.
- [x] You've added file named `shivam-jha2712.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/shivam-jha2712.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "![Screenshot (1298)](https://user-images.githubusercontent.com/71783722/193400453-e7624e82-9d78-45b1-80e1-d20876e46ea5.png)",
```
Comment (krshkodes) [inline profiles/shivam-jha2712.json:2]:
```suggestion
  "image": "https://user-images.githubusercontent.com/71783722/193400453-e7624e82-9d78-45b1-80e1-d20876e46ea5.png",
```

**Review by krshkodes — CHANGES_REQUESTED** [review]
I believe it is meant to be the url and not the markdown

**Review by shivam-jha2712 — COMMENTED** [review]
I have completed the review could you just look into it once

File: profiles/shivam-jha2712.json (hunk 1 above)
Comment (shivam-jha2712) [reply profiles/shivam-jha2712.json]: Could you please let me know if I should go for a pull request again by making the changes or how I should edit the current pull request if YES? How to do so

Comment (krshkodes) [reply profiles/shivam-jha2712.json]: I think you did it right 👍

**Review by krshkodes — APPROVED** [review]
Looks good to me!

**Comment by shivam-jha2712** [comment]
> Looks good to me!

Could you please merge it then

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**